test/helpers/testHelper.js

```diff
@@ -0,0 +1,30 @@
+const fs = require('fs');
+const path = require('path');
+const { Client } = require('cql-translation-service-client');
+
+const TRANSLATION_SERVICE_URL = 'http://localhost:8080/cql/translator';
+
+const client = new Client(TRANSLATION_SERVICE_URL);
+
+/**
+ * Translate all cql
+ * @returns {Object} ELM from translator
+ */
+async function translateCQL() {
+  const cqlPath = path.resolve(path.join(__dirname), '../../src');
+  const cqlFiles = fs.readdirSync(cqlPath).filter((f) => path.extname(f) === '.cql');
+  const cqlRequestBody = {};
+
+  cqlFiles.forEach((f) => {
+    cqlRequestBody[path.basename(f, '.cql')] = {
+      cql: fs.readFileSync(path.join(cqlPath, f), 'utf8'),
+    };
+  });
+
+  const elm = await client.convertCQL(cqlRequestBody);
+  return elm;
+}
+
+module.exports = {
+  translateCQL,
+};
```

Comment:
It looks like this only gets used by the `build.js` script. Could we merge the two files? Or are we anticipating future components that will utilize this function and shouldn't know about the `build.js` script?